src/RestWrite.js

```diff
@@ -1104,6 +1102,29 @@ RestWrite.prototype.sanitizedData = function() {
   return Parse._decode(undefined, data);
 }
 
+// Returns an updated copy of the object
+RestWrite.prototype.buildUpdatedObject = function (extraData) {
+  var updatedObject = triggers.inflate(extraData, this.originalData);
```

Comment:
I was thinking that the style requirement of `let` instead of `var` was only for declarations inside blocks, not at the beginning of a function. Is that wrong ?